### ISSUE TITLE
Remove Bottom Navigation Bar on certain routes/pages

### ISSUE THREAD

**Issue by mfenz** [bizz84/nested-navigation-demo-flutter#11]
Hi! Thank you for the great plugin. 

Is it possible to remove the bottom navigation on certain pages? 
i.e. I want to open some fullscreen pages without the bottom navigation bar and return to pages with the bottom navigation bar once the fullscreen page finished (like Android's startActivityForResult). 

Thank you!

**Comment by Mickey-A-Mouse** [comment]
Hello)) 

I think, you don't need to remove it) Actually you could use the first level navigator, by using Navigator.of(context).push(..);

**Comment by Ali-hd** [comment]
> Hello))
> 
> I think, you don't need to remove it) Actually you could use the first level navigator, by using Navigator.of(context).push(..);

Hi, I'm facing the same problem and I don't understand your answer. I'm trying to create a full screen page ontop of the bottom nav bar. I tried Navigator.of(context).push(new route) but it doesn't work.

**Comment by Mickey-A-Mouse** [comment]
Sorry, I've just updated my answer! So, what I mean by using first navigator:

SplashScreen(no bottom navBar) -> MainScreen(with app bar);

MainScreen screens:

HomeScreen -> ProductScreen -> etc.

ProfileScreen -> Navigator.of(context, rootNavigator: true); -> Login screen(no bottom navBar)

Is that ok? More info: https://stackoverflow.com/a/55218112/10084055

**Comment by Ali-hd** [comment]
> Sorry, I've just updated my answer! So, what I mean by using first navigator:
> 
> SplashScreen(no bottom navBar) -> MainScreen(with app bar);
> 
> MainScreen screens:
> 
> HomeScreen -> ProductScreen -> etc.
> 
> ProfileScreen -> Navigator.of(context, rootNavigator: true); -> Login screen(no bottom navBar)
> 
> Is that ok? More info: https://stackoverflow.com/a/55218112/10084055

Thanks!! Its working just by adding rootnavigator: true.

**Comment by Mickey-A-Mouse** [comment]
This solution works only with 2 level navigation (root, custom). But I don't know how to access to the middle navigator for 3 or more level navigations(root, middle, custom).
